DEVELOPMENT.md

```diff
@@ -79,5 +79,36 @@ Look at the logs in the Log Group `/aws/sagemaker/studio` and the Log Stream `$d
 
 Ensure your SageMaker Studio execution role has permissions to pull the image from ECR. See the [ECR documentation](https://docs.aws.amazon.com/AmazonECR/latest/userguide/security_iam_id-based-policy-examples.html) for IAM permissions required to pull an image.
 
+### Files With Unsupported UIDs/GIDs
 
+SageMaker only supports UIDs/GIDs less than 65535. You should ensure that your container does not have any layers containing a file with a GID/UID higher than 65535.
 
+You can check if you have any files in your Docker image that exceed 65535 by adding the following RUN commands at the end of your Dockerfile
+```
+RUN echo $(sudo find / -gid +65535 -ls 2>/dev/null)
+RUN echo $(sudo find / -uid +65535 -ls 2>/dev/null)
+```
+or the following commands in a terminal inside the container
+```
+$ echo $(sudo find / -gid +65535 -ls 2>/dev/null)
+$ echo $(sudo find / -uid +65535 -ls 2>/dev/null)
+```
+If you find any output from the above commands, you have 2 options:
+
+1. Pinpoint the step in your Dockerfile that creates the files shown in the output append the following to the RUN command in your Dockerfile as shown:
```

Comment:
Let's clarify that this needs to be added to each RUN command in the Dockerfile.

Reply:
moved the part about adding these run commands after each run command further above. it should be more clear now